build/tasks/build.js

```diff
@@ -81,15 +81,6 @@ gulp.task('build-system', ['build-html-system'], function () {
     .pipe(gulp.dest(paths.output + 'system'));
 });
 
-gulp.task('build-dts', function(){
```

Comment:
Assuming removing this is because there is / was a duplicate task with same name?